x-pack/platform/test/functional/apps/maps/group4/file_upload/geojson.js

```diff
@@ -63,7 +63,9 @@ export default function ({ getPageObjects, getService }) {
       await retry.try(async () => {
         await maps.waitForLayersToLoad();
         const tooltipText = await maps.getLayerTocTooltipMsg(indexName);
-        expect(tooltipText).to.be(`${indexName}\nFound ~281 documents. This count is approximate.`);
+        expect(tooltipText).to.contain(`${indexName}`);
+        expect(tooltipText).to.match(/Found ~\d+ documents/);
```

Comment:
I think we should understand why the number is changing before committing this change. If the browser size is the same and the file is the same, then it should be uploaded the same each time.